src/pages/saksbehandling/vilkårsOppsummering/VilkårsOppsummering.tsx

```diff
@@ -0,0 +1,111 @@
+import { Systemtittel, Undertittel } from 'nav-frontend-typografi';
+import React from 'react';
+
+import VilkårvurderingStatusIcon from '~components/VilkårvurderingStatusIcon';
+import { mapToVilkårsinformasjon, vilkårTittelFormatted } from '~features/saksoversikt/utils';
+import { useI18n } from '~lib/hooks';
+import { Nullable } from '~lib/types';
+import { Behandlingsinformasjon } from '~types/Behandlingsinformasjon';
+import { SøknadInnhold } from '~types/Søknad';
+import { Vilkårtype, VilkårVurderingStatus } from '~types/Vilkårsvurdering';
+
+import { FaktablokkTitteltype } from '../steg/faktablokk/faktablokker/faktablokkUtils';
+import FastOppholdFaktablokk from '../steg/faktablokk/faktablokker/FastOppholdFaktablokk';
+import FlyktningFaktablokk from '../steg/faktablokk/faktablokker/FlyktningFaktablokk';
+import FormueFaktablokk from '../steg/faktablokk/faktablokker/FormueFaktablokk';
+import LovligOppholdFaktablokk from '../steg/faktablokk/faktablokker/LovligOppholdFaktablokk';
+import PersonligOppmøteFaktablokk from '../steg/faktablokk/faktablokker/PersonligOppmøteFaktablokk';
+import UførhetFaktablokk from '../steg/faktablokk/faktablokker/UførhetFaktablokk';
+import UtenlandsOppholdFaktablokk from '../steg/faktablokk/faktablokker/UtenlandsOppholdFaktablokk';
+
+import messages from './vilkårsOppsummering-nb';
+import styles from './vilkårsOppsummering.module.less';
+
+const VilkårsOppsummering = (props: {
+    søknadInnhold: SøknadInnhold;
+    behandlingsinformasjon: Behandlingsinformasjon;
+}) => {
+    const intl = useI18n({ messages });
+    const vilkårsinformasjon = mapToVilkårsinformasjon(props.behandlingsinformasjon);
+
+    return (
+        <div>
+            <Systemtittel className={styles.tittel}>{intl.formatMessage({ id: 'page.tittel' })}</Systemtittel>
+            <div className={styles.vilkårsblokkerContainer}>
+                {vilkårsinformasjon.map((v) => (
+                    <VilkårsBlokk
+                        key={v.vilkårtype}
+                        tittel={vilkårTittelFormatted(v.vilkårtype)}
+                        status={v.status}
+                        fraSøknadComponent={mapVilkårtypeToFaktablokk(v.vilkårtype, props.søknadInnhold)}
+                        begrunnelse={v.begrunnelse}
+                    />
+                ))}
+            </div>
+        </div>
+    );
+};
+
+const VilkårsBlokk = (props: {
+    status: VilkårVurderingStatus;
+    tittel: string;
+    fraSøknadComponent: Nullable<JSX.Element>;
```

Comment:
Ikke helt sikker på vad `fraSøknadComponent` är? Men det ser ut som det är det søkaren har fyllt in i søknaden, men isåfall ska den ju ikke kunne vara nullable eller?